ws-client/src/transport.rs

```diff
@@ -227,7 +233,24 @@ impl<'a> WsTransportClientBuilder<'a> {
 					match self.mode {
 						Mode::Plain => TlsOrPlain::Plain(socket),
 						Mode::Tls => {
-							let connector = async_tls::TlsConnector::default();
+							let mut client_config = rustls::ClientConfig::default();
+							if let Some(path) = &self.custom_certificate {
+								// TODO: How to give use async for file io and still give add_pem_file() a std::io::BufRead?
+								let f = std::fs::File::open(&path)?;
+								let mut reader = io::BufReader::new(f);
+								let (added, failed) = client_config
+									.root_store
+									.add_pem_file(&mut reader)
```

Comment:
I realized that another solution is to use `rustls_native_certificates` instead such as [hyper-rustls does](https://github.com/ctz/hyper-rustls/blob/master/src/connector.rs#L34-#L42)